MainRepository/Program.cs

```diff
@@ -2,14 +2,19 @@
 {
     private static void Main(string[] args)
     {
-        if (int.TryParse(args[0], out int n) && n > 0 && n <= 25) Console.WriteLine($"The {n}th armstrong number is {Armstrong(n)}");
-        else Console.WriteLine("Enter the valid input");
+        try

```

Comment:
Alignment is completely wrong. Use "Ctrl + K" & "Ctrl + F" to view the difference.
And I'm unable to compile.